tests/Unit/RpcLoggerTest.php

```
@@ -354,10 +357,357 @@ public function testLogWithComplexContext(): void
+    }
+
+    public function testCustomProcessorIntegration(): void
+    {
+        // Create a custom processor for email addresses
+        $emailProcessor = new class implements ObjectProcessor {
+            public function canProcess(mixed $value): bool
+            {
+                return \is_string($value) && \filter_var($value, FILTER_VALIDATE_EMAIL) !== false;
+            }
+
+            public function process(mixed $value, callable $processor): mixed
```

PHPMD (2.15.0)

Avoid unused parameters such as '$processor'. (undefined)

(UnusedFormalParameter)